cdk/visit/lambda_code/submit_quiz/submit_quiz.py

```diff
@@ -0,0 +1,84 @@
+import json
+import boto3
+from boto3.dynamodb.conditions import Key
+import os
+import time
+from typing import Tuple
+
+
+class SubmitQuizFunction():
```

Comment:
Nit but should we name this to something more generic like QuizFunction? Down the line we might want to add Read. Update, List, Delete APIs and it would make sense to have them all live in the same function

Reply:
You are right. We are adding a Get request next and it would make it easier to put it in one function than multiple lambdas. So rename it to QuizFunction then quiz or user_quiz?